### DISCUSSION TITLE
I am not able to run any of the example in simulator

### DISCUSSION THREAD

**Discussion by poa010101** [NVIDIA/NVFlare#1896] [Q&A] [answered]
root@222a27042f33:/home/jay/app/NVFlare# nvflare simulator NVFlare/examples/hello-world/hello-numpy-sag -w /simulator-example/ -c client0,client1,client2,client3 -t 1
2023-08-02 20:10:56,907 - SimulatorRunner - ERROR - Simulator setup error: RuntimeError: ValueError: App hello-numpy-sag in deploy_map doesn't exist for job hello-numpy-sag
2023-08-02 20:10:56,908 - MPM - INFO - MPM: Good Bye!

**Comment by YuanTingHsieh** [comment, marked as answer]
You should pass in the job folder to nvflare simulator command.
So please try the following:

``nvflare simulator NVFlare/examples/hello-world/hello-numpy-sag/jobs/hello-numpy-sag/ -w /simulator-example/ -c client0,client1,client2,client3 -t 1``

**Reply by YuanTingHsieh** [reply]
@poa010101

**Reply by poa010101** [reply]
Got it work.thanks
